### ISSUE TITLE
Broken Github-Actions

### ISSUE THREAD

**Issue by chriz-uniba** [inveniosoftware/flask-security-fork#58]
Package version (if known): Current master (2023-10-04)

## Describe the bug

After making a minor change in the `setup.cfg`-file the github-actions do not run through. I think the error is not related to the changes that were made (see https://github.com/inveniosoftware/flask-security-fork/pull/57)

```
Tests (3.8, pypi)
failed Oct 4, 2023 in 48s

Run ./run-tests.sh
lists of files in version control and sdist match

Configuration error:
There is a programmable error in your configuration file:

Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/sphinx/config.py", line 332, in eval_config_file
    exec(code, namespace)
  File "/home/runner/work/flask-security-fork/flask-security-fork/docs/conf.py", line 18, in <module>
    from flask_security import __version__
  File "/home/runner/work/flask-security-fork/flask-security-fork/flask_security/__init__.py", line 29, in <module>
    from flask_login import login_required
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/flask_login/__init__.py", line 12, in <module>
    from .login_manager import LoginManager
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/flask_login/login_manager.py", line 33, in <module>
    from .utils import _create_identifier
  File "/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/flask_login/utils.py", line 14, in <module>
    from werkzeug.urls import url_decode
ImportError: cannot import name 'url_decode' from 'werkzeug.urls' (/opt/hostedtoolcache/Python/3.8.18/x64/lib/python3.8/site-packages/werkzeug/urls.py)

Error: Process completed with exit code 2.
```

## Expected behavior

github-actions should run through for the master-branch

**Comment by yashlamba** [comment]
Hey @chriz-uniba, this is related to dependency pinning. We didn't have an upper cap on Flask and Werkzeug versions, both of which release 3.x.x recently.

This probably will be fixed with #56

**Comment by chriz-uniba** [comment]
yes - I rebased and the tests are happy now. Thanks.
